### PR TITLE
Elasticsearch: add additional settings section

### PR DESCRIPTION
**Pull request by matyax**
Following up with the configuration page refactors, this PR adds an [Additional settings](https://docs.google.com/document/d/11XMaYHSMSra8AoN2-yVW-5QPbjEXgkAFbTIF6N84cGE/edit#heading=h.g16g42msnv4g) section.

**Which issue(s) does this PR fix?**:

Part of https://github.com/grafana/grafana/issues/67383

**Special notes for your reviewer:**

Before:

imagen

After:

imagen

### CODE REVIEW

**Comment by matyax** [comment]
FYI @gelicia @josiahg

File: public/app/plugins/datasource/elasticsearch/configuration/DataLinks.tsx [hunk 1]
```diff
@@ -29,60 +33,63 @@ export const DataLinks = (props: Props) => {
   const styles = useStyles2(getStyles);
 
   return (
-    <>
-      <h3 className="page-heading">Data links</h3>
+    <ConfigSubSection
+      title="Logs"
```
Comment (gwdawson) [inline public/app/plugins/datasource/elasticsearch/configuration/DataLinks.tsx:37]: this this supposed to be "Data links" instead of logs?

Comment (matyax) [reply public/app/plugins/datasource/elasticsearch/configuration/DataLinks.tsx:37]: Yes! Good catch 🤝

**Review by itsgareth — APPROVED** [review]
LGTM!

**Pull request merged**